### PR TITLE
refactor: improve logging

### PR DESCRIPTION
**Pull request by Mini256**

### CODE REVIEW

File: deploy/Makefile [hunk 1]
```diff
@@ -3,17 +3,17 @@ SHELL := /bin/bash
 bots-namespace:
 	kubectl apply -f cluster/bots_namespace.yaml
 
-app-config:
-	kubectl apply -f config/app_config.yaml
+ti-sync-bot-config:
+	kubectl apply -f config/ti_sync_bot_config.yaml -n ti-community-bots
```
Comment (hi-rustin) [inline deploy/Makefile:7]: Please put namespace in yaml and not in the command.

**Review by hi-rustin — CHANGES_REQUESTED** [review]

File: deploy/Makefile (hunk 1 above)
Comment (hi-rustin) [reply deploy/Makefile:7]: We don't seem to need to maintain these configurations in this repository, do we?

File: src/events/common.ts [hunk 2]
```diff
@@ -85,14 +85,21 @@ export async function getPullRequestPatch(
...
+  try {
+    patchResponse = await github.pulls.get({
+      ...pullKey,
+      headers: {
+        Accept: "application/vnd.github.VERSION.patch",
+      },
+    });
+  } catch (err) {
+    github.log.error("Failed to get patch file of pull request", err);
+    console.error(err);
```
Comment (hi-rustin) [inline src/events/common.ts:99]: Don't use console.error.

File: deploy/Makefile (hunk 1 above)
Comment (Mini256) [reply deploy/Makefile:7]: Prod configuration can be managed uniformly in configs repo, but as a project, it should also include complete files including deployment files, similar to the configuration in tichi's prow-dev.

File: src/events/common.ts [hunk 3]
```diff
@@ -85,14 +85,21 @@ export async function getPullRequestPatch(
...
+  }
 
-  if (patchResponse.status === 200) {
+  if (patchResponse && patchResponse.status === 200) {
```
Comment (hi-rustin) [inline src/events/common.ts:102]: Maybe patchResponse?.status?

**Review by hi-rustin — CHANGES_REQUESTED** [review]

File: deploy/Makefile (hunk 1 above)
Comment (hi-rustin) [reply deploy/Makefile:7]: Maybe we could create a dev folder, as I think it might be complicated if each bot maintains these deployment files separately.

Comment (hi-rustin) [reply deploy/Makefile:7]: Since we may have multiple bots, it would be very complicated if we need to clone and deploy them separately.

Comment (Mini256) [reply deploy/Makefile:7]: I thought for a while, you are right.

**Review by hi-rustin — APPROVED** [review]

**Pull request merged**